Dockerfile

```diff
@@ -14,7 +14,18 @@ RUN if [ "$SKIP_TESTS" = 'false' ]; then \
   npm install cspell && npm run spellcheck; fi
 
 #-----------------------------
-FROM otomi/tools:1.4.9 as prod
+FROM otomi/tools:1.4.9 as test
+
+ENV APP_HOME=/home/app/stack
+RUN mkdir -p $APP_HOME
+WORKDIR $APP_HOME
+
+COPY . .
+
+RUN bats bin/tests
+
+#-----------------------------
+FROM otomi/tools:1.4.8 as prod
```

Comment:
can this be 1.4.9 also then?

Reply:
Yes actually I want to merge the "aws" tools image and then bump this one.